virttest/utils_misc.py

```diff
@@ -2765,20 +2747,56 @@ def get_all_disks_did(session, partition=False):
     :return: a dict with all disks did lists each include disk
              kname, serial and wwn.
     """
```

Comment:
The doc string should be updated to the right one, it's about get disks did one.